tests/codegen/call_ntop_char16.cpp

```diff
@@ -40,39 +39,6 @@ declare void @llvm.lifetime.end.p0i8(i64, i8* nocapture) #1
 attributes #0 = { nounwind }
 attributes #1 = { argmemonly nounwind }
 )EXPECTED");
-#else
-R"EXPECTED(; Function Attrs: nounwind
-declare i64 @llvm.bpf.pseudo(i64, i64) #0
-
-; Function Attrs: argmemonly nounwind
-declare void @llvm.lifetime.start.p0i8(i64, i8* nocapture) #1
-
-define i64 @"kprobe:f"(i8* nocapture readnone) local_unnamed_addr section "s_kprobe:f_1" {
-entry:
-  %"@x_key" = alloca i64, align 8
-  %inet = alloca [20 x i8], align 4
-  %1 = getelementptr inbounds [20 x i8], [20 x i8]* %inet, i64 0, i64 0
-  call void @llvm.lifetime.start.p0i8(i64 -1, i8* nonnull %1)
-  store i32 10, [20 x i8]* %inet, align 4
-  %2 = getelementptr inbounds [20 x i8], [20 x i8]* %inet, i64 0, i64 4
-  %probe_read = call i64 inttoptr (i64 4 to i64 (i8*, i64, i8*)*)(i8* %2, i64 16, i64 0)
-  %3 = bitcast i64* %"@x_key" to i8*
-  call void @llvm.lifetime.start.p0i8(i64 -1, i8* nonnull %3)
-  store i64 0, i64* %"@x_key", align 8
-  %pseudo = call i64 @llvm.bpf.pseudo(i64 1, i64 1)
-  %update_elem = call i64 inttoptr (i64 2 to i64 (i8*, i8*, i8*, i64)*)(i64 %pseudo, i64* nonnull %"@x_key", [20 x i8]* nonnull %inet, i64 0)
-  call void @llvm.lifetime.end.p0i8(i64 -1, i8* nonnull %3)
-  call void @llvm.lifetime.end.p0i8(i64 -1, i8* nonnull %1)
-  ret i64 0
-}
-
-; Function Attrs: argmemonly nounwind
-declare void @llvm.lifetime.end.p0i8(i64, i8* nocapture) #1
-
-attributes #0 = { nounwind }
-attributes #1 = { argmemonly nounwind }
-)EXPECTED");
-#endif
```

Comment:
I think the codegen is equal across LLVM versions with those changes. Let's see what CI says.